packages/kit/src/packaging/index.js

```diff
@@ -144,11 +153,44 @@ export async function make_package(config, cwd = process.cwd()) {
 		const full_path = path.join(cwd, pathname);
 		if (fs.lstatSync(full_path).isDirectory()) continue; // just to be sure
 
-		const package_path = path.join(cwd, config.kit.package.dir, pathname);
+		const package_path = path.join(abs_package_dir, pathname);
 		if (!fs.existsSync(package_path)) fs.copyFileSync(full_path, package_path);
 	}
 }
 
+/**
+ * Resolves the `$lib` alias.
+ *
+ * TODO: make this more generic to also handle
```

Comment:
this is a bit of an unexpected place to wrap the line. should probably have more like 80 or 100 characters on this line before wrapping